### ISSUE TITLE
Did something hapenned? Not working anymore

### ISSUE THREAD

**Issue by osaulonery** [exile-center/better-trading#154]
Extension not showing on trade site and disabled for some reason.

**Comment by dbjorge** [comment]
We're not aware of any issues like this. Could you please share:

- Exact version of Better PathOfExile Trading shown on about://extensions (eg, "1.8.0")
    - If about://extensions shows any Errors for the extension, please copy/paste them here
- Whether you have any other extensions installed that might also be interacting with the trade site (eg, TFT's)
    - If so, whether the issue still occurs when you disable them on the about://extensions page
- Exact browser and version from about://settings/help (eg, "Google Chrome Version 131.0.6778.205 (Official Build) (64-bit)")
- An example of an exact trade site URL you're visiting where you're not seeing the extension as expected (eg, "https://www.pathofexile.com/trade2/search/poe2/Standard/pqkMa9ZS0")
- A description/screenshot of exactly what you mean by "not showing and disabled" - does the trade site look completely unmodified, or is the right panel for the extension appearing as empty, or something else?
- Any error messages in the F12 developer tools console when you visit the trade site page in question

**Comment by dbjorge** [comment]
For you and anyone else seeing this: it's definitely possible that this sort of issue could be caused by a bug in the extension, but another possible cause is visiting a fake phishing version of the trade site (there is at least one of these going around with a google ad campaign right now). If you end up on a fake trade site with a non-official URL, the extension won't show up there. Double check that the URL you're visiting really starts with https://pathofexile.com.

**Comment by Anceph** [comment]
I had the same problem and figured out that the extension only works if the URL has "www" at the front. For example, "https://www.pathofexile.com/trade2/search/poe2/Standard/8eJ9q32sV" works but "https://pathofexile.com/trade2/search/poe2/Standard/8eJ9q32sV" doesn't. I checked extension's site access and it only includes "*://poe.ninja/*" and "*://www.pathofexile.com/*". That's probably why.

**Comment by travismathews** [comment]
Yep, if i visit www it works again.  Whats odd, is i have had the trade site bookmarked, and it just suddenly stopped working., no change to the bookmark.

**Comment by jsgueglia84** [comment]
Is there a way to give the extension access to just pathofexile.com without the "www" it's very annoying for me since I open many tabs frequently and just type "p" in the address bar for efficiency.

**Comment by dbjorge** [comment]
Fixed by 2959587441ac3b5bfe824387e0d48dc628b5740c , will be released as 1.9.1 once it passes chrome webstore review.
